modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1268,6 +1268,58 @@ definition {
 		}
 	}
 
+	@description = "LPS-165994 - Verify that the user cannot insert an invalid date on Start / End fields when creating a filter"
+	@priority = "5"
+	test CannotInsertInvalidDate {
+		property portal.acceptance = "true";
```

Comment:
Delete it. Because will be a low priority.

Reply:
Not sure what I needed to remove here, but the comment above, I changed the priority.

Reply:
No worries, removed acceptance and changed priority back to 5.